silkworm/core/state/intra_block_state.cpp

```diff
@@ -301,6 +301,17 @@ void IntraBlockState::set_storage(const evmc::address& address, const evmc::byte
     journal_.emplace_back(new state::StorageChangeDelta{address, key, prev});
 }
 
+evmc::bytes32 IntraBlockState::get_transient_storage(const evmc::address& addr, const evmc::bytes32& key) {
+    return transient_storage_[addr][key];
```

Comment:
This is "minimal" implementation. But we probably don't want to create map entries on every access for null value. Not sure about address though.

Reply:
This is not a problem?

Reply:
It's probably fine. I'd keep it simple for now. If it turns out to be a bottleneck in performance profiling, we can optimize this later.